Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -152,6 +152,100 @@ public void ProcArgs_CmdLinePropertiesOverrideFileSettings()
             AssertExpectedValue(SonarProperties.HostUrl, "http://host", processedArgs);
         }
 
+        [TestMethod]
```

Comment:
General comment for all the changes. I think we should use `SonarCloud` not `Sonarcloud` in the names of the tests, properties,...

Reply:
I used `Sonarcloud` because the argument is named that way (sonar.scanner.sonarcloudUrl).

Reply:
isn't this a typo in the property name? since Url starts with capital letter, I think cloud should do the same.

Reply:
It (probably) is because of the environment variable naming convention described in the spec:
sonar.scanner.sonarcloudUrl -> SONAR_SACNNER_SONARCLOUD_URL